lib/patches/json_api_client/resource.rb

```diff
@@ -11,15 +12,8 @@ def save
         else
           self.class.requestor.create(self)
         end
-
         if last_result_set.has_errors?
-          last_result_set.errors.each do |error|
-            if error.source_parameter
-              errors.add(error.source_parameter, error.title || error.detail)
-            else
-              errors.add(:base, error.title || error.detail)
-            end
-          end
+          fill_errors
```

Comment:
Our patch needed updating to use the `json_api_client` gem's latest method of dealing with errors